sound/soc/sof/loader.c

```diff
@@ -235,6 +235,13 @@ static int xman_get_cc_info(struct snd_sof_dev *sdev,
 	return 0;
 }
 
+static int xman_get_probe_info(struct snd_sof_dev *sdev,
+			       struct xman_elem_header *hdr)
+{
+	/* todo: */
```

Comment:
?? if this does nothing then remove this patch, or add the functionality.

Reply:
removed